File: android/app/src/main/java/top/ourfor/app/iPlayClient/module/ModuleManager.java
package top.ourfor.app.iPlayClient.module;

import androidx.annotation.NonNull;
import com.facebook.react.ReactPackage;
import com.facebook.react.bridge.NativeModule;
import com.facebook.react.bridge.ReactApplicationContext;
import com.facebook.react.uimanager.ViewManager;

import java.util.ArrayList;
import java.util.List;

public class ModuleManager implements ReactPackage {

    @Override
    public List<NativeModule> createNativeModules(ReactApplicationContext reactContext) {
        List<NativeModule> nativeModules = new ArrayList<>();
        nativeModules.add(new IntentModule(reactContext));
        nativeModules.add(new FontModule(reactContext));
        return nativeModules;
    }

    @NonNull
    @Override
    public List<ViewManager> createViewManagers(@NonNull ReactApplicationContext reactApplicationContext) {
        List<ViewManager> viewManagers = new ArrayList<>();
        viewManagers.add(new PlayerViewManager(reactApplicationContext));
        viewManagers.add(new DemoViewManager(reactApplicationContext));
        return viewManagers;
    }
}

File: android/app/src/main/java/top/ourfor/app/iPlayClient/ModuleManager.kt
package top.ourfor.app.iPlayClient

import com.facebook.react.ReactPackage
import com.facebook.react.bridge.NativeModule
import com.facebook.react.bridge.ReactApplicationContext

class ModuleManager : ReactPackage {

    override fun createViewManagers(
        reactContext: ReactApplicationContext
    ) = listOf(
        PlayerViewManager(reactContext)
    ).toMutableList()

    override fun createNativeModules(
        reactContext: ReactApplicationContext
    ): MutableList<NativeModule> = listOf(
        IntentModule(reactContext)
    ).toMutableList()
}